### PR TITLE
Add support for feature restore files from backup

### PR DESCRIPTION
**Pull request by svt-pchavan**
Signed-off-by: pratiksha chavan <pratiksha.chavan@hpe.com>

### Description
Add support for feature restore files from backup

### Issues Resolved
None

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Comment by svt-pchavan** [comment]
Updated the content-type to minimum version from were the API is supported

File: examples/backups.py [hunk 1]
```diff
@@ -168,3 +172,16 @@
...
+
+vm = machines.get_by_name(test_vm_name)
+backup = vm.create_backup("backup_test_from_sdk_" + str(time.time()))
+print(f"{backup}")
+print(f"{pp.pformat(backup.data)} \n")
+try:
+    print("\n\nbackup restore files")
+    target_vm = machines.get_by_name(target_vm_name)
+    target_vm_id = target_vm.data["id"]
+    backup.restore_files(target_vm_id, paths)
```
Comment (sijeesh) [inline examples/backups.py:193]: I think we should give this feature to the vm resource also to restore from its backup.
Eg. vm.restore(back_up_name) 

What's your opinion on this?

**Review by sijeesh — CHANGES_REQUESTED** [review]

File: examples/backups.py (hunk 1 above)
Comment (svt-pchavan) [reply examples/backups.py:193]: Yes but the only thing is, there is no direct API for restore functionality under VM resource. We have '/backups/{bkpId}/restore' under backup resource which is already automated.

**Comment by sijeesh** [comment]
@svt-pchavan , please resolve the conflicts.

**Review by sijeesh — APPROVED** [review]
LGTM

**Pull request merged**